### PR TITLE
sql-math-function-power

### LINKED ISSUE #2219: [Term Entry] SQL Math Functions POWER
### Reason/inspiration (optional)

We would like a new term entry for the Math Function, `POWER`, in SQL.

The entry should go in a file in docs/content/sql/concepts/math-functions/terms/power/power.md

The entry should: 

- Explain what the math function `POWER` is in SQL and what it’s used for, starting with an introductory paragraph (2-3 sentences).
- Include SQL database compatibility as a note.
- Links out to entries as needed.

Please refer to the [entry templates (term entry template in particular)](https://github.com/Codecademy/docs/blob/main/documentation/), [content standards](https://github.com/Codecademy/docs/blob/main/documentation/content-standards.md), and [markdown style guide](https://curriculum-documentation.codecademy.com/content-guidelines/markdown-style-guide/) when working on the PR for this issue.

### Entry Type (select all that apply)

New Term for Existing Concept Entry

### Code of Conduct

- [X] By submitting this issue, I agree to follow Codecademy Doc's [Code of Conduct](https://github.com/Codecademy/docs/blob/main/.github/CODE_OF_CONDUCT.md).

### For Maintainers

- [X] Labels added
- [x] Issue is assigned

### PR DESCRIPTION
**Pull request by nikolaspoczekaj**
### Description

This closes #2219 . Thanks for assigning me.

### Type of Change

- Adding a new entry

### Checklist

- [x] All writings are my own.
- [x] My entry follows the Codecademy Docs style guide.
- [x] My changes generate no new warnings.
- [x] I have performed a self-review of my own writing and code.
- [x] I have checked my entry and corrected any misspellings.
- [x] I have made corresponding changes to the documentation if needed.
- [x] I have confirmed my changes are not being pushed from my forked `main` branch.
- [x] I have confirmed that I'm pushing from a new branch named after the changes I'm making.
- [x] Under "Development" on the right, I have linked any issues that are relevant to this PR (write "Closes #<issue number> in the "Description" above).

### CODE REVIEW

File: content/sql/concepts/math-functions/terms/power/power.md [hunk 1]
```diff
@@ -0,0 +1,50 @@
+---
+Title: 'POWER()'
+Description: 'Calculates the power of a number.'
```
Comment (caupolicandiaz) [inline content/sql/concepts/math-functions/terms/power/power.md:3]:
```suggestion
Description: 'Returns the value of a number raised to the power of another number.'
```

**Review by caupolicandiaz — CHANGES_REQUESTED** [review]
@nikolaspoczekaj Just completed my initial review and I only had a couple of comments. Please update and tag me when you've finished. Thanks

**Comment by nikolaspoczekaj** [comment]
@caupolicandiaz Done :)

**Review by caupolicandiaz — APPROVED** [review]
@nikolaspoczekaj Looks good

**Review by SSwiniarski — APPROVED** [review]
@nikolaspoczekaj Looks good to merge.

**Comment by SSwiniarski** [comment]
@nikolaspoczekaj, congrats on the new entry. Unfortunately it won't be visible on the site until we merge [2228](https://github.com/Codecademy/docs/pull/2228).

**Comment by yangc95** [comment]
🥳 @nikolaspoczekaj livelink: https://www.codecademy.com/resources/docs/sql/math-functions/power

Screenshot 2023-05-25 at 3 28 59 PM

**Pull request merged**